py3status/modules/file_status.py

```diff
@@ -1,22 +1,50 @@
 # -*- coding: utf-8 -*-
 """
-Display if a file or directory exists.
+Display if files or directories exists.
 
 Configuration parameters:
-    cache_timeout: how often to run the check (default 10)
-    format: format of the output. (default '{icon}')
-    icon_available: icon to display when available (default '●')
-    icon_unavailable: icon to display when unavailable (default '■')
-    path: the path to a file or dir to check if it exists (default None)
-
-Color options:
-    color_bad: Error or file/directory does not exist
-    color_good: File or directory exists
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module
+        (default '\?color=paths [\?if=paths ●|■]')
```

Comment:
What is the goal? For readability? Compliance?

Reply:
I don't remember all the details. I think readability so some things are not rendered as blocks or to avoid cause minor confusion. It may or may not had to do more with control characters.

```
FILE_SEPARATOR = u' '
GROUP_SEPARATOR = u' '
RECORD_SEPARATOR = 'u ' 
UNIT_SEPARATOR = u' '
```
versus more explicit...
```
FILE_SEPARATOR = u'\u231a'
GROUP_SEPARATOR = u'\u231d'
RECORD_SEPARATOR = u'\u231e' 
UNIT_SEPARATOR = u'\u231f'
```